WordPress/Classes/ViewRelated/Gutenberg/GutenbergMediaPickerHelper.swift

```diff
@@ -1,7 +1,9 @@
 import Foundation
 import CoreServices
-import WPMediaPicker
+import UIKit
```

Comment:
Xcode added the imports automatically, and then I tried to order them by starting with core libraries.